updates for joint compilation with pfnet

ttinoco opened the pull request:
- eliminated compilation warnigns due to unused variables
- changed TRUE/FALSE to LF_TRUE/LF_FALSE

We will have issues with line endings (windows vs linux). I think there are some git settings to handle these discrepancies:

https://stackoverflow.com/a/40821931

ttinoco committed:
updates for joint compilation with pfnet

dmitry-shchetinin commented:
I did the merge and added .gitattributes to the root of my local repository to make sure that the line endings on Github will be Linux-style. I also updated the release files in case you want to use LineFlow as a standalone on Windows.

ttinoco commented:
Very cool!

On Jan 26, 2018 6:36 AM, "dmitry-shchetinin" <notifications@github.com>
wrote:

> I did the merge and added .gitattributes to the root of my local
> repository to make sure that the line endings on Github will be
> Linux-style. I also updated the release files in case you want to use
> LineFlow as a standalone on Windows.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dmitry-shchetinin/LineFlow/pull/1#issuecomment-360800798>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGFYf0IhMW_NmsXFsFQ5icBuGcKCwct_ks5tOeLhgaJpZM4RsVSz>
> .
>
